integration/typegen-test.ts

```diff
@@ -30,15 +30,19 @@ const viteConfig = tsx`
   };
 `;
 
-const assertType = tsx`
-  export function assertType<T>(t: T) {}
```

Comment:
If typegen was broken there were no type errors since `skipLibCheck: true` was configured for the test fixture. `skipLibCheck: true` does make sense to avoid random type errors from dependencies. However, this caused all types from typegen to be `any`.

Previously, the `assertType` check was too loose and didn't throw any type errors when `any` was passed in. The new approach is more robust and does correctly throw type errors for `any`.